docs/application/dotnet/guides/nui/text.md

```diff
@@ -150,6 +150,15 @@ To align the text in a text label:
 
 
 
+<a name="encoding"></a>
+### Text property can accept UTF16 or UTF32 character encoding

```

Comment:
How about simplifying the title, and adding one simple description?
For example

\### UTF16 or UTF32 character encoding

The text property can accept UTF16 or UTF32 character encoding